setup.py

```diff
@@ -16,7 +16,7 @@
     version=versioneer.get_version(),
     cmdclass=versioneer.get_cmdclass(),
     description="Modin: Make your pandas code run faster by changing one line of code.",
-    packages=find_packages(),
+    packages=find_packages(exclude=["scripts", "scripts.*"]),
```

Comment:
Even though our "scripts" folder is used for testing only, should we move it under "modin" directory so "scripts" is accessible via "modin.scripts"?

Reply:
That is another option, I agree. But I don't see much value in that.

Reply:
We may put there something useful in future so that might be accessible when installing Modin package. Does that make sense?

Reply:
In this case we'll move `scripts` when needed, and this exclude won't prevent it from inclusion 😉

Reply:
Yes, this is a one line change rather than a refactor, so I am in favor of this change first

Reply:
Okay, makes sense.